components/blog/Post.tsx

```diff
@@ -0,0 +1,87 @@
+import Image from 'next/image';
+import { getStrapiMedia, formatDate } from '../../utils/blog/api-helpers';
+import { postRenderer } from '../../utils/blog/post-rendered';
+
+export interface Article {
+	id: number;
+	attributes: {
+		title: string;
+		description: string;
+		slug: string;
+		cover: {
+			data: {
+				attributes: {
+					url: string;
+				};
+			};
+		};
+		authorsBio: {
+			data: {
+				attributes: {
+					name: string;
+					avatar: {
+						data: {
+							attributes: {
+								url: string;
+							};
+						};
+					};
+				};
+			};
+		};
+		blocks: any[];
+		publishedAt: string;
+	};
+}
+
+const Post: React.FC<{ article: Article["attributes"] }> = ({ article }) => {
+	const { title, description, publishedAt, cover, authorsBio, blocks } = article;
+	const author = authorsBio.data?.attributes;
+	const imageUrl = getStrapiMedia(cover.data?.attributes.url);
+	const authorImgUrl = getStrapiMedia(authorsBio.data?.attributes.avatar.data.attributes.url);
+
+	if(imageUrl === null || authorImgUrl === null) {
+		console.error(`[blog/Post] imageUrl = ${imageUrl}, authorImgUrl = ${authorImgUrl}`);
+	}
+	// TODO - add placeholder image and/or author image
```

Comment:
Ye aise hi chhodne wale ho abhi ke liye?
Aage ka code phat nahi jayega?

Reply:
Fat jayega as in if the url is null then kuch nahi dikhega, baki sab sahi rehta hai